packages/flame/lib/src/game/flame_game.dart

```diff
@@ -220,12 +201,6 @@ class FlameGame<W extends World> extends ComponentTreeRoot
         Duration.microsecondsPerSecond;
   }
 
-  @override
-  Projector get viewportProjector => oldCamera.viewport;
```

Comment:
do still have anything that uses this interface Projector?

Reply:
Not after this PR